stuff/main.go

```diff
@@ -102,7 +101,7 @@ func main() {
 		"Cache templates. False for online editing while development.")
 	staticResource := flag.String("staticdir", "static",
 		"Directory with static resources")
-	port := flag.Int("port", 2000, "Port to serve from")
+	listenAddress := flag.String("web.listen-address", ":2000", "Port to serve from")
```

Comment:
I'd call it --bind-address for less confusion.

Reply:
But it's called `ListenAndServeTLS(addr, ...)`, hence listen address. :smile: